test/hoverable.test.js

```diff
@@ -0,0 +1,50 @@
+/*global describe, afterEach, it*/
+'use strict';
+
+var expect = require('expect.js');
+
+var oHoverable = require('./../main.js');
+
+describe('o-hoverable', function() {
+
+	var testHoverable;
+
+	afterEach(function() {
+		testHoverable.destroy();
+	});
+
+	it('should initialise the module', function() {
+		expect(window.document.body.hasAttribute('data-o-hoverable--js')).to.be(false);
+		testHoverable = new oHoverable();
+		expect(window.document.body.hasAttribute('data-o-hoverable--js')).to.be(true);
+	});
+
+	it('should destroy the module', function() {
```

Comment:
I link the second one, it's what I started doing in o-gallery and will probably do it from now onwards.